source/installing.rst

```diff
@@ -19,11 +19,11 @@ Like many other |percona| products, |pdp| is installed by using the
    <https://www.percona.com/doc/percona-repo-config/percona-release.html#percona-release-update-latest-version>`_
    of |percona-release| for details.
 
-As soon as |percona-release| is up-to-date, *set up* the |pdp| product (``ppg-11``).
+As soon as |percona-release| is up-to-date, *set up* the |pdp| product (``ppg-12``).
 
 .. code-block:: bash
 
-   $ sudo percona-release setup ppg11
+   $ sudo percona-release setup ppg12
```

Comment:
just a question.
We are going to issue ppg-12.2 but why we do not show how to install this distribution but generic only?

Reply:
This comes from that we have special repo for each distribution and that is why this format doesn't suit for distributions